cmd/clusterctl/hack/create-local-repository.py

```diff
@@ -148,6 +151,7 @@ def create_local_repositories():
 
         repo = p.get('repo', '.')
         config_folder = p.get('configFolder', 'config/default')
+        metadata_file = repo+'/metadata.yaml'
```

Comment:
I did something a little different. No changes to the clusterctl-settings file, but instead just use the repo's metadata.yaml. So now...if you put `infrastructure-aws`, it will load its metadata.yaml. And if you put `infrastructure-azure`, it will load the corresponding metadata.yaml. 🙂 

Don't know why I didn't think of this before! 🙄  Great idea @fabriziopandini!